contrib/ci/Jenkinsfile

```diff
@@ -1,27 +1,27 @@
 pipeline {
```

Comment:
What is the reason to reformat this file to use tabs over space?

Reply:
Before seems weird formatted with 4 space steps?

Reply:
https://gluon.readthedocs.io/en/v2017.1.x/dev/basics.html#development-guidelines says to use tabs instead of spaces.

Reply:
Ok.